pep-0001.txt

```diff
@@ -230,28 +238,40 @@ PEP Review & Resolution
 
 Once the authors have completed a PEP, they may request a review for
 style and consistency from the PEP editors.  However, the content and
-final acceptance of the PEP must be requested of the BDFL, usually via
-an email to the python-dev mailing list.  PEPs are reviewed by the
-BDFL and his chosen consultants, who may accept or reject a PEP or
-send it back to the author(s) for revision.  For a PEP that is
-predetermined to be acceptable (e.g., it is an obvious win as-is
-and/or its implementation has already been checked in) the BDFL may
-also initiate a PEP review, first notifying the PEP author(s) and
+final acceptance of the PEP must be requested of the core developers, usually
+via an email to the python-dev mailing list.  PEPs are reviewed by the
+core developers, who may accept or reject a PEP or send it back to the
+author(s) for revision.
+
+For a PEP that is predetermined to be acceptable (e.g., it is an obvious win
+as-is and/or its implementation has already been checked in) the Steering
+Council may also initiate a PEP review, first notifying the PEP author(s) and
 giving them a chance to make revisions.
 
-The final authority for PEP approval is the BDFL. However, whenever a new
-PEP is put forward, any core developer that believes they are suitably
+The final authority for PEP approval is the Steering Council. However, whenever
+a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the BDFL's delegate (or "PEP czar") for that PEP. If their self-nomination
-is accepted by the other core developers and the BDFL, then they will have
-the authority to approve (or reject) that PEP. This process happens most
-frequently with PEPs where the BDFL has granted in principle approval for
-*something* to be done, but there are details that need to be worked out
-before the PEP can be accepted.
+the Steering Council's delegate for that PEP.
+
+If their self-nomination is accepted by the other core developers and the
```

Comment:
Perhaps:

...self-nomination has broad consensus of the other core developers and the acceptance of the Steering Council, they ...

Reply:
Perhaps we should turn that around, and instead of "broad consensus", operate on a veto basis.  E.g. someone self-nominates for BDFL-delegate.  Other core devs can petition the SC to veto that delegation.  If they concur, then a new delegate must be found.  If not, then the original delegate is approved.  I.e. only the SC can veto a delegate self-nomination, but core devs can petition the SC for that ruling if they think something is amiss.

Reply:
I almost changed this to assume consensus in the absence of a Steering Council veto to bring it closer to the wording in https://www.python.org/dev/peps/pep-0013/#membership (just without the complexity of core devs having to formally vote on every BDFL-Delegate appointment), and only left it alone on the basis of minimalism.

I'll go ahead and make that change now, since it isn't just me that found the current wording underspecified :)